docs/en/guides/pipeline-integrations/administration/advanced-toolkit-administration.md

```diff
@@ -0,0 +1,1526 @@
+
+In the example above, `./tank` is starting a Python shell for you and setting up a few variables that you'll need. This obviates a bit of groundwork on your part, such as setting up authentication and user sessions and starting the connection to your {% include product %} site, by providing these handles and contexts in the above variables:
+
+- The `tk` variable has a handle that will let you access the [Toolkit API](https://developer.shotgridsoftware.com/tk-core/core.html?highlight=context#the-toolkit-core-api).
+- The `{% include product %}` variable has a handle to the [{% include product %} API](https://developer.shotgridsoftware.com/python-api/) that's already connected to your site and is ready to query the database and other things in your [{% include product %} site].
+- The context variable reads your current context (Project, Shot, Asset, Pipeline Step, User, etc.). [Here is an example of an active context](https://developer.shotgridsoftware.com/tk-core/core.html?highlight=context#sgtk.Context). In this example, a context instance is used to collect a set of key fields describing the current context. We sometimes refer to the context as the current work area. Typically this would be the current Shot or Asset that someone is working on. You can see in this example that the Project is "Chasing the Light", the id is 1184, and the Shot is "moo87". There are no Pipeline Steps or Tasks assigned, but the user is Manne. You may need to know this information when writing scripts for the Toolkit API, and everything is available in this handy context variable:
+
+```python
+<Sgtk Context:   Project: {'type': 'Project', 'id': 88, 'name': 'Chasing the Light'}
+     Entity: {'type': 'Shot', 'id': 1184, 'name': 'moo87'}
+     Step: None
+     Task: None
+     User: {'type': 'HumanUser', 'id': 42, 'name': 'Manne \xc3\x96hrstr\xc3\xb6m'}
+     Additional Entities: []>
+```
+
+The `engine` variable points to the currently running instance of the [shell engine](https://developer.shotgridsoftware.com/2ad59ee8/?title=Shell).
+
+### dump_config
+
+Available in core `v0.18`.
+
+Dumps an existing environment configuration to `STDOUT` or to another file. Has the ability to dump a full or sparse representation of the config. A full representation includes default values for all settings, while a sparse representation only includes values that differ from the setting's default. The output of the command by default includes comments for each setting that note the manifest where the default value can be found and the default value if it differs from what is in the configuration. The command does not allow dumping an environment config to an existing file as a safety precaution.
+
+Usage:
+
+```
+./tank dump_config env_name [--sparse | --full] [--no_debug-comments] [--file=/path/to/output/file.yml]
+```
+
+Example output from this command:
+
+```python
+> ./tank dump_config shot_step --sparse --file=/tmp/shot_step.yml
+
+Welcome to the Shotgun Pipeline Toolkit!
```

Comment:
Should be `Welcome to SGTK!`

Reply:
Fixed in latest commit, thanks @mathurf !